capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -116,7 +116,7 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
      * @return returns List<RestaurantPageViews> with the PageView list inside each instance
      *         of RestaurantPageViews sorted by week and year in ascending order.
      */
-    public List<RestaurantPageViews> getAllPageViews() {
+    public List<RestaurantPageViews> getAllPageViews(String identificationType) {
```

Comment:
Please update the javadoc to explain what identificationType is.

What are all the identification types? Also, should this live in an enum?

Reply:
Fixed javadoc.